### PR TITLE
Adding a simple sample application for 1st round demo

### PR DESCRIPTION
**Pull request by mpeters**

### CODE REVIEW

File: docker/sample-application/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.7.8-alpine3.12
```
Comment (font) [inline docker/sample-application/Dockerfile:1]: I realize this is a sample app, but we should try to use `registry.access.redhat.com/ubi8/python-38` even though it's not as lean, if we end up using this app more.

Comment (mpeters) [reply docker/sample-application/Dockerfile]: Good point. I'll have to get used to checking registry.access.redhat.com for images first before I go to dockerhub.

Comment (mpeters) [reply docker/sample-application/Dockerfile]: Done

File: docker/sample-application/app.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+def hello_world():
+    pod_ip = os.getenv('POD_IP')
+    if pod_ip is None:
+        pod_ip = '<unknown>'
+
+    node_ip = os.getenv('NODE_IP')
+    if node_ip is None:
+        node_ip = '<unknown>'
+
+    return 'Howdy folks! 👍\nPOD IP: %s\nNODE IP: %s\n' % (pod_ip, node_ip)
```
Comment (font) [inline docker/sample-application/app.py:16]: :+1:

**Pull request merged**